source/main/gui/panels/GUI_GameAbout.cpp

```diff
@@ -131,18 +130,20 @@ void CLASS::initMisc()
 	AuthorsText = AuthorsText + color1 + "cptf:" + white + " Several Linux gcc fixes" + newline;
 	AuthorsText = AuthorsText + color1 + "88Toyota:" + white + " Clutch force patches" + newline;
 	AuthorsText = AuthorsText + color1 + "synthead:" + white + " Minor Linux fixes" + newline;
-	AuthorsText = AuthorsText + color1 + "ulteq:" + white + " sound engine, lots of fixes" + newline;
-	AuthorsText = AuthorsText + color1 + "theshark:" + white + " various fixes" + newline;
+	AuthorsText = AuthorsText + color1 + "ulteq:" + white + " Sound engine, lots of fixes" + newline;
+	AuthorsText = AuthorsText + color1 + "theshark:" + white + " Various fixes" + newline;
+	AuthorsText = AuthorsText + color1 + "skybon:" + white + " Fixes" + newline;
 	AuthorsText = AuthorsText + color1 + "Clockwork (a.k.a VeyronEB):" + white + " GUI Designer and tweaker" + newline;
 	AuthorsText = AuthorsText + color1 + "Klink:" + white + " Terrains conversion, few fixes and tweaks, dashboard designer" + newline;
 	AuthorsText = AuthorsText + color1 + "hagdervriese:" + white + " Linux fixes" + newline;
+	AuthorsText = AuthorsText + color1 + "AnotherFoxGuy:" + white + " Various fixes" + newline;
 
 	//Core Content Contributors
 	AuthorsText = AuthorsText + newline;
 	AuthorsText = AuthorsText + orange + "Core Content Contributors:" + newline;
 	AuthorsText = AuthorsText + color1 + "donoteat:" + white + " Improved spawner models, terrain work" + newline;
-	AuthorsText = AuthorsText + color1 + "kevinmce:" + white + " old Character" + newline;
-	AuthorsText = AuthorsText + color1 + "09Challenger:" + white + " new character animations" + newline;
```

Comment:
Both `kevinmce` and `09Challenger` are also mentioned here: https://github.com/RigsOfRods/rigs-of-rods/blob/028518296ae867f0edc3267a153725bd0b76cc67/AUTHORS.md

I think we should leave them in. What does @only-a-ptr say?